extensions/wikia/Email/Controller/ForgotPasswordController.class.php

```diff
@@ -14,75 +14,52 @@
  * @package Email\Controller
  */
 class ForgotPasswordController extends EmailController {
+	private $tempPass;
 
 	public function initEmail() {
```

Comment:
You should probably just view the full file for this one, since it really should appear as a new file anyway.  Since this was the first example controller, there were tons of things that changed.